hello.py

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env python3
 print ("hello world")
-print("HOLA")
+print("HOLA")
```

Comment:
You can use a only language, spanish or english

Reply:
it's in english now